pareto/other_models/CM_module/operational/run_optimal_desal.py

```diff
@@ -0,0 +1,74 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021-2024 by the software owners:
+# The Regents of the University of California, through Lawrence Berkeley National Laboratory, et al.
+# All rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the U.S.
+# Government consequently retains certain rights. As such, the U.S. Government has been granted for
+# itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license in
+# the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit others to do so.
+#####################################################################################################
+
+
+"""
+Determining Optimal Desalination Location Run
+This file looks to find the optimal location of the desalination facility within the network.
+
+This file runs the model in the following order:
+1. Run for treatment site attached for different nodes in the network
+2. Display cost results plots
+3. Display cost summary for best network
+"""
+import pyomo.environ as pyo
+from pareto.utilities.get_data import get_data
+from pareto.utilities.cm_utils.gen_utils import report_results_to_excel
+from pareto.utilities.cm_utils.data_parser import data_parser
+from pareto.utilities.cm_utils.run_utils import (
+    node_rerun,
+    print_results_summary,
+)
+from importlib import resources
+
+from set_param_list import set_list, parameter_list
+
+# -------------------------Loading data and model------------------------------
+"""
+Purpose of each of the files:
+large_case_study_Li.xlsx: This file is the large permian case study with no TDS concentrations.
+small_case_study_Li.xlsx: This file is the small permian case study with LI and TDS concentrations.
+"""
+
+fname = "small_case_study"
+with resources.path(
+    "pareto.case_studies",
+    "small_case_study_Li.xlsx",
+) as fpath:
+    [df_sets, df_parameters] = get_data(fpath, set_list, parameter_list)
+
+
+# checking for tabs in the excel sheet
```

Comment:
```suggestion
```
There seems to be no relevant code underneath this comment - can the comment be removed?